app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -36,29 +31,31 @@ fun Movie(
             .clickable { onMovieCardClick() },
         shape = RoundedCornerShape(10.dp)
     ) {
-        AsyncImage(
-            model = movie.imageUrl,
-            contentDescription = null,
-            modifier = Modifier.clip(RoundedCornerShape(10.dp)),
-            contentScale = ContentScale.Crop,
-        )
-        FavoriteButton(
-            isFavorite = movie.isFavorite,
-            modifier = Modifier.padding(5.dp),
-            onFavoriteButtonClick = onFavoriteButtonClick
-        )
+        Box() {
```

Comment:
why did you wrapped this with Box()?

Reply:
Because without it, when i create MovieCards in HomeScreen, the FavoriteButton takes up the whole card, like in the image. I don't know how to correctly do it without the box. Could you give me some pointers on a better way to do it?

![Screenshot_2022-11-23-21-40-12-948_agency five codebase android movieapp](https://user-images.githubusercontent.com/93868899/203642682-6a8a290d-3aab-427d-8b55-9e7e6905f536.jpg)

Reply:
That's okay :) thanks for the explanation